### PR TITLE
Desktop portfolio version

### PR DESCRIPTION
**Pull request by Nicolaswg**
*The main idea of this project is:*

1. Build the Desktop version of our portfolio
2. Make use of new CSS syntax (Media Query)
3. CSS animation
4. RWD design

*Project achievements*

- RWD design. 
- Query syntax
- Animation
🦅

### CODE REVIEW

File: query.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    margin-top: 40px;
+  }
+
+  .about-project {
+    display: flex;
+  }
+
+  .about-items-a {
+    flex-grow: 1;
+    width: 50%;
```
Comment (lily-coder) [inline query.css]: - [ ] [REQUIRED] Please ensure that this section, the left section of the (About Myself section) takes a width of 40% so as to match the original design. Yours is 50%

File: query.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  }
+
+  .about-items-b {
+    flex-grow: 1;
+    width: 50%;
```
Comment (lily-coder) [inline query.css]: - [ ] [REQUIRED] Please ensure that this section, the right section of the (the About Myself section) takes a width of 60% so as to match the original design. Yours is 50%

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @Nicolaswg , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ Good commit messages
✔️ Good job with the linter checks
✔️ Correct Github flow was used

There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Proofread your portfolio to get rid of spelling errors. This is very important as hiring managers looking at your portfolio in the future may take that as you are not keen on small details, which is something very vital for a software developer.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alonsofalconi — APPROVED** [review]
Hi @Nicolaswg,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**